dovpanda/core.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
 from dateutil.parser import parse
 from dovpanda import base, config
-from dovpanda.base import Ledger
+from dovpanda.base import Ledger, MAX_CSV_SIZE
+from os.path import getsize
```

Comment:
import `os` and call the full path `os.path.getsize`

Reply:
why?

Reply:
I want a clean name space

Reply:
still unchanged

Reply:
I still don't get what's the difference between
`from dateutil.parser import parse`
and
from os.path import getsize